libs/race/src/lib/component/display-bid/bid/compare/compare-player-bid.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, forwardRef, input } from '@angular/core';
+import { AbstractControlComponent } from '../../../../../../../control/src/lib/abstract-control-component';
```

Comment:
Der er problemer her med import. Udover det så er jeg ikke sikker på jeg synes det er en control. Kunne det ikke bare være et `output` Så var der ikke noget behov for at nedarve fra en abstract control der ikke er tilgængelig